distribution/docs/src/main/jdocs/content/_security-framework/security-encryption-api-contents.adoc

```diff
@@ -0,0 +1,18 @@
+:title: Security Encryption API
+:type: subSecurityFramework
+:status: published
+:parent: Security Encryption
+:order: 00
+:summary: Security Encryption API.
+
+The Security Encryption API bundle provides the framework for the encryption service.
+Applications that use the encryption service should import this bundle and use the interfaces defined within it instead of calling an implementation directly.
```

Comment:
❓ Classes and packages can be imported, but not I'm not sure bundles can be imported.

Reply:
Does it work to say "encryption service should use the interfaces defined within it..."